avocado/utils/cpu.py

```diff
@@ -97,28 +97,20 @@ def get_cpu_arch():
     """
     Work out which CPU architecture we're running on
     """
+    cpu_table = [('^cpu.*(RS64|POWER3|Broadband Engine)', 'power'),
+                 ('^cpu.*POWER4', 'power4'),
+                 ('^cpu.*POWER5', 'power5'),
+                 ('^cpu.*POWER6', 'power6'),
+                 ('^cpu.*POWER7', 'power7'),
+                 ('^cpu.*POWER8', 'power8'),
+                 ('^cpu.*PPC970', 'power970'),
```

Comment:
OK, no problem, we are but humans and the biggest problem (probably in the world) is that one bases his answer on his understanding of other person's loose representation of his view. And then the whole process happens again at the other side...

Never mind, so far I came up to decision of the professor Higgins from My fair lady who says that he treats everyone the same and that is important. I share the opinion and I try to be the same to everyone (which does not necessary mean kind to everyone, just consistent...)